zebedee-cms/src/main/java/com/github/onsdigital/zebedee/Zebedee.java

```diff
@@ -16,11 +16,7 @@
 import com.github.onsdigital.zebedee.model.encryption.EncryptionKeyFactory;
 import com.github.onsdigital.zebedee.model.publishing.PublishedCollections;
 import com.github.onsdigital.zebedee.permissions.service.PermissionsService;
-import com.github.onsdigital.zebedee.service.DatasetService;
-import com.github.onsdigital.zebedee.service.ImageService;
-import com.github.onsdigital.zebedee.service.KafkaService;
-import com.github.onsdigital.zebedee.service.ServiceStore;
-import com.github.onsdigital.zebedee.service.ServiceStoreImpl;
+import com.github.onsdigital.zebedee.service.*;
```

Comment:
Can you update your ide settings to not user wildcard imports please.

Reply:
🙈 Thanks, have set up a new mac and haven't got the settings right again.